src/confdb/confdb_setup.c

```diff
@@ -44,6 +44,35 @@
      "\n"
 
 
+errno_t confdb_read_ini(TALLOC_CTX *mem_ctx,
+                     const char *config_file,
+                     const char *config_dir,
+                     bool allow_missing_file,
+                     struct sss_ini **_ini)
+{
+    int ret;
+
+    *_ini = sss_ini_new(mem_ctx);
+    if (*_ini == NULL) {
+        return ENOMEM;
+    }
+
+    ret = sss_ini_read_sssd_conf(*_ini,
+                                 config_file,
+                                 config_dir);
```

Comment:
This can maybe go to single line and still fit characters limit?

Reply:
Done.